shared/elements/gaia-header/dist/gaia-header.js

```diff
@@ -613,11 +693,11 @@ const MINIMUM_FONT_SIZE_CENTERED = 20;
  * This is the minimum font size that we can take
  * when the header title is not centered in the window.
  */
-const MINIMUM_FONT_SIZE_UNCENTERED = 18;
+const MINIMUM_FONT_SIZE_UNCENTERED = 16;
```

Comment:
ahah yeah https://github.com/mozilla-b2g/gaia/pull/30264 never landed :) we landed the change in the v2.2 branch only.